AsyncDisplayKit/Details/ASTableLayoutController.m

```diff
@@ -64,6 +64,17 @@ - (NSSet *)indexPathsForScrolling:(ASScrollDirection)scrollDirection rangeMode:(
   return indexPaths;
 }
 
+- (void)allIndexPathsForScrolling:(ASScrollDirection)scrollDirection rangeMode:(ASLayoutRangeMode)rangeMode displaySet:(NSSet **)displaySet preloadSet:(NSSet **)preloadSet
+{
+    if (displaySet == NULL || preloadSet == NULL) {
+    return;
+  }
+
+  *displaySet = [self indexPathsForScrolling:scrollDirection rangeMode:rangeMode rangeType:ASLayoutRangeTypeDisplay];
+  *preloadSet = [self indexPathsForScrolling:scrollDirection rangeMode:rangeMode rangeType:ASLayoutRangeTypePreload];
```

Comment:
Of course, this method could have a more efficient union implementation as well.  But I figured this could wait for the new async flow layout, and as-implemented it would be tough since we need the frames of each item to decompose them from the unioned rect.  It may not even make sense to do here because this class is pretty fast at generating the paths for each rect uniquely.